core/primitives/src/stateless_validation/state_witness.rs

```diff
@@ -19,87 +18,31 @@ use near_schema_checker_lib::ProtocolSchema;
 
 /// Represents max allowed size of the raw (not compressed) state witness,
 /// corresponds to the size of borsh-serialized ChunkStateWitness.
-pub const MAX_UNCOMPRESSED_STATE_WITNESS_SIZE: ByteSize =
-    ByteSize::mib(if cfg!(feature = "test_features") { 512 } else { 64 });
+pub const MAX_UNCOMPRESSED_STATE_WITNESS_SIZE: u64 =
+    ByteSize::mib(if cfg!(feature = "test_features") { 512 } else { 64 }).0;
```

Comment:
@pugachAG Do we still needs this?

Reply:
if you mean the `test_features` part then yes, let's keep it